mariadb/entrypoint.sh

```diff
@@ -128,5 +128,10 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 		fi
 	fi
 fi
-
-docker-entrypoint.sh $@
\ No newline at end of file
+# On first install cp the init.sh
+if [ ! -e "/docker-entrypoint-initdb.d/.init" ]; then
```

Comment:
Hi @db260179,

the `docker-entrypoint-initdb.d` directory isn't a volume, so docker will re-run this at every restart. The `mariadb` image  already runs the scripts inserted in `/docker-entrypoint-initdb.d/` once. So you don't need to check this here.